edk2toollib/utility_functions_test.py

```diff
@@ -0,0 +1,24 @@
+# @file utility_functions_test.py
+# unit test for locate_tools module.  Only runs on Windows machines.
+#
+#
+# Copyright (c) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+import unittest
+import sys
+import os
+import edk2toollib.utility_functions as utilities
+
+
+class UtilityFunctionsTest(unittest.TestCase):
+
+    def test_RunPythonScript(self): # simple run yourself!
```

Comment:
should we add a test case that uses all named parameters

Reply:
Added a few more test cases